synapse/handlers/saml_handler.py

```diff
@@ -84,6 +86,25 @@ def __init__(self, hs: "synapse.server.HomeServer"):
         # a lock on the mappings
         self._mapping_lock = Linearizer(name="saml_mapping", clock=self._clock)
 
+    def _render_error(
+        self, request, error: str, error_description: Optional[str] = None
+    ) -> None:
+        """Renders the error template and respond with it.
+
+        This is used to show errors to the user. The template of this page can
+        be found under ``synapse/res/templates/sso_error.html``.
```

Comment:
Just checking this wasn't a c/p from RST error?

I typically use single backticks in docstrings, but you might prefer it this way? In which case it's fine :)